app/assets/stylesheets/application.css.scss

```diff
@@ -65,6 +59,18 @@ legend {
 
   .button {
     background-color: rgb(89, 84, 84);
+    border: 0;
+    box-shadow: inset 0 0 0 1px rgb(255, 255, 255);
+    font-weight: 100;
+    letter-spacing: 1px;
+    transition: all 0.5s ease 0s;
+    opacity: 1;
+
+    &:hover {
+      background-color: #9dbcbc;
```

Comment:
Color literals like `#9dbcbc` should only be used in variable declarations; they should be referred to via variable everywhere else.